packages/lodestar-cli/src/testnets/index.ts

```diff
@@ -20,6 +20,17 @@ export function getTestnetConfig(testnet: TestnetName): IBeaconNodeOptionsPartia
   }
 }
 
+export function getTestnetParamsUrl(testnet: TestnetName): string | null {
+  switch (testnet) {
+    case "altona":
+      return "https://raw.githubusercontent.com/eth2-clients/eth2-testnets/master/shared/altona/config.yaml";
```

Comment:
I don't see `DEPOSIT_CHAIN_ID` in this link, probably it prevents me from doing a deposit
```
Starting 1 deposits
 ✖ Error: chainId address mismatch (argument="transaction", value={"to":"0x1234567890123456789012345678901234567890","gasLimit":400000,"value":{"_hex":"0x01bc16d674ec800000","_isBigNumber":true},"data":"0x22895118000000000000000000000000000000000000000000000000000000000000008000000000000000000000000000000000000000000000000000000000000000e000000000000000000000000000000000000000000000000000000000000001203a11b4afbcb000d77cb24e6e14d87c962965fa9632f59dc2502ddecc05a4bcd4000000000000000000000000000000000000000000000000000000000000003091aadbb5fcc07a2215e7056847f07aaa1fff03ead480d5b5da45c3d58c03c72f863ff50447cf670d0ea93eabad77a33d00000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000002000cfa79e4e42951cfbb687df5916899a8d15fdf05d33b0f01d84dd7ac38d1d5200000000000000000000000000000000000000000000000000000000000000608288fa3b2a74f3ad18ba6adfc451a2a34c70aa5f7b47075a51cbf8f48a4f693408b22b8cc7a6728871e45433866d2d26187f4ba7975a59c9053ba28a34d374c12844dfe1444b7c6ec0fe5e09c36683e0d389cfd2a52e8ba599a8a753947be9f5","chainId":1}, code=INVALID_ARGUMENT, version=abstract-signer/5.0.1)
```

Reply:
failed to get medalla deposit events too, maybe it's the same error
```
(node:36549) UnhandledPromiseRejectionWarning: Error: could not detect network (event="noNetwork", code=NETWORK_ERROR, version=providers/5.0.2)
```
I found that chainId in eth1 provider is 1